hack/generated/apis/microsoft.resources/v1alpha1api20200601/resourcegroup_types.go

```diff
@@ -127,6 +137,54 @@ func (status *ResourceGroupStatus) PopulateFromARM(owner genruntime.KnownResourc
 	return nil
 }
 
+func (status *ResourceGroupStatus) ConvertStatusTo(destination genruntime.ConvertibleStatus) error {
+	dst, ok := destination.(*ResourceGroupStatus)
+	if !ok {
+		return errors.Errorf("cannot convert ResourceGroupStatus to %T", destination)
+	}
+
+	dst.ID = status.ID
+	dst.Name = status.Name
+	dst.Location = status.Location
+	dst.ManagedBy = status.ManagedBy
+	dst.ProvisioningState = status.ProvisioningState
+
+	dst.Tags = make(map[string]string)
+	for k, v := range status.Tags {
+		dst.Tags[k] = v
+	}
+
+	for _, c := range status.Conditions {
+		dst.Conditions = append(dst.Conditions, c.Copy())
+	}
+
+	return nil
+}
+
+func (status *ResourceGroupStatus) ConvertStatusFrom(source genruntime.ConvertibleStatus) error {
+	src, ok := source.(*ResourceGroupStatus)
+	if !ok {
+		return errors.Errorf("cannot convert ResourceGroupStatus from %T", source)
```

Comment:
Same comment as below about this log seeming backwards to me?

Should it instead read: `cannot convert from %T to ResourceGroupStatus`?

Reply:
Again, the error message matches the implementation.

We're converting from the provided instance (`source`) to the current one (`status`).